package.json

```diff
@@ -45,11 +46,6 @@
     "gulp-nsp": "^2.1.0"
   },
   "repository": "mateusmedeiros/generator-prelude",
-  "scripts": {
-    "link": "npm run prepublish && sudo npm link",
-    "prepublish": "cp -R generators generatorsSrc && gulp publish",
```

Comment:
`npm link` is essential to develop any yeoman generator, because it makes it very easy to test the changes without having to publish the package.

The `cp` command on the `prepublish` is because `gulp publish` will transform the js files with babel. I made it like that so that you can take a look at the source and see the correct generator structure (generators directory with app and other subgenerators directories inside) but at the same time publish a transformed tree to npm.

When publishing, the `generators` directory will be copied to `generatorsSrc`, and then gulp will run babel through it and output it into a `generators` directory. Npm will publish the package, and afterwards the `generators` directory will be removed and the `generatorsSrc` renamed back to `generators`.

The `link` and `unlink` scripts are just simple wrappers that will run these things before and after `npm link`, just like they would be run when trully publishing the package.